src/Pages/Main/Components/LimitedEvent/LimitedEvent.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import ProductCard from '../ProductCard/ProductCard';
+import './LimitedEvent.scss';
+
+class LimitedEvent extends Component {
+  constructor() {
+    super();
+    this.state = {
+      time: '',
+    };
+  }
+
+  componentDidMount() {
+    this.setTimer();
+  }
+
+  componentWillUnmount() {
+    this.setTimer();
+  }
+
+  setTimer = () => {
+    const x = setInterval(() => {
+      let countdownDate = new Date('Apr 24, 2021, 00:00:00').getTime();
+      let now = new Date().getTime();
+      let remaining = countdownDate - now;
```

Comment:
```suggestion
      let remainingTime = countdownDate - now;
```
남아 있는게 제품 수량인지 시간인지 보다 명확하면 좋겠습니다

Reply:
수정하였습니다!